### PR TITLE
[F] Improvements to external video external_id

### LINKED ISSUE #1950: Add instructions to resource external video ID field to explain how to find the ID
Per #1949, users aren't sure how to get the ID. Assigning to @krosemichael to provide us with exact language for the instructions.

### PR DESCRIPTION
**Pull request by SMaxOwok**
Closes #1950

### CODE REVIEW

File: client/src/backend/components/resource/form/kind/Video.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ export default class ResourceFormKindVideo extends PureComponent {
           label="Video ID"
           name="attributes[externalId]"
           placeholder="Video ID"
+          instructions={
+            'The video ID is located in the URL after "?v=" in YouTube video URLs, and after "Vimeo.com/" in Vimeo URLs.'
```
Comment (zdavis) [inline client/src/backend/components/resource/form/kind/Video.js]: Please lowercase "Vimeo" in "Vimeo.com"

File: client/src/backend/components/resource/form/kind/__tests__/__snapshots__/Video-test.js.snap [hunk 2]
```diff
@@ -153,6 +153,11 @@ exports[`Backend.Resource.Form.Video component when sub kind is external_video r
         type="text"
         value=""
       />
+      <span
+        className="instructions"
+      >
+        The video ID is located in the URL after "?v=" in YouTube video URLs, and after "Vimeo.com/" in Vimeo URLs.
```
Comment (zdavis) [inline client/src/backend/components/resource/form/kind/__tests__/__snapshots__/Video-test.js.snap]: Please lowercase "Vimeo" in "Vimeo.com"

File: client/src/backend/components/resource/form/kind/Video.js (hunk 1 above)
Comment (zdavis) [reply client/src/backend/components/resource/form/kind/Video.js]: Change to:
```
The video's ID is located after "?v=" in YouTube URLs, and after "vimeo.com/" in Vimeo URLs.
```

**Comment by SMaxOwok** [comment]
Is there a better way to do the erroring in the model/service?

Would it be better to return nil and check that?

**Pull request merged**